pkg/account/downloader/groups.go

```diff
@@ -0,0 +1,197 @@
+/*
+ * @license
+ * Copyright 2023 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package downloader
+
+import (
+	"context"
+	accountmanagement "github.com/dynatrace/dynatrace-configuration-as-code-core/gen/account_management"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/log"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/account"
+	"github.com/google/uuid"
+	"strings"
+)
+
+type (
+	Groups []group
+
+	levelID = string
+	group   struct {
+		group         *account.Group
+		dto           *accountmanagement.GetGroupDto
+		permissionDTO *accountmanagement.PermissionsGroupDto
+		bindings      map[levelID]*accountmanagement.LevelPolicyBindingDto
+	}
+)
+
+func (a *Account) Groups(policies Policies, tenants Environments) (Groups, error) {
```

Comment:
Are these exported methods a leftover or does my IDE wrongly not find them used?
It looks like "Account.DownloadConfiguration" is the only entry point that is being used?